cmd/octorpki/filter.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "github.com/cloudflare/gortr/prefixfile"
+
+func FilterBad(roalist []prefixfile.ROAJson) ([]prefixfile.ROAJson, error) {
```

Comment:
Can we rename this to `FilterInvalidPrefixLen` or something? :-)

Reply:
Not in this pr. As said in the description: Just moving existing code here.

Reply:
Well, okay. I took the time now to clean this up a bit.

Reply:
Grazie